consultation_analyser/consultations/download_consultation.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+from django.forms.models import model_to_dict
+
+from consultation_analyser.consultations.public_schema import (
+    ConsultationWithResponses,
+)
+
+
+def select_keys_from_model(model, keys):
+    model_attrs = model_to_dict(model)
+    return {key: model_attrs[key] for key in keys}
+
+
+def consultation_to_json(consultation):
```

Comment:
Does this method output themes? I don't think it does.

Reply:
Next PR!